src/Homepage/Homepage.js

```diff
@@ -1,44 +1,98 @@
+import BrandHeadline from '../common/BrandHeadline'
 import BulletList from '../common/BulletList'
 import Button from '../common/Button'
+import Card from '../common/Card'
+import Collapse from '../common/Collapse'
 import FloatingButtonContainer from '../common/FloatingButtonContainer'
 import HeadlineUnderline from '../common/HeadlineUnderline'
-import TextLink from '../common/TextLink'
 
 export default function () {
   return (
     <>
       <img alt="" src="/img/mirror.svg" />
+      <Logo alt="Youlity" src="/img/logo.svg" />
       <HeadlineUnderline>
-        <h1>Entschlüsseln Sie Ihre Persönlichkeit!</h1>
+        <h1>Discover Your Personality!</h1>
       </HeadlineUnderline>
-      <BulletList listType="bullet_circle">
-        <li>
-          Informieren Sie sich über persönliche Eigenarten, die Ihr Denken,
-          Fühlen und Handeln prägen.
-        </li>
-        <li>
-          Entwickeln Sie ein besseres Verständnis für sich selbst in
-          Beziehungen.
-        </li>
-        <li>
-          Profitieren Sie von speziell auf Sie zugeschnittenen Alltagstipps.
-        </li>
-      </BulletList>
+      <p>
+        Youlity ist eine Plattform, die Ihnen hilft, Ihre eigene Persönlichkeit
+        über sogenannte Persönlichkeitsstile besser zu verstehen.
+      </p>
+      <Collapse headline="Was sind Persönlichkeitsstile?">
+        <p>
+          Die Psychologie hat sich lange hauptsächlich mit
+          Persönlichkeits-störungen beschäftigt. Doch auch gesunde Menschen
+          haben ein Interesse daran, Ihre Persönlichkeit zu verstehen, und in
+          bestimmten Situationen ihr Handeln zu verändern. Dazu dient unser
+          Youlity-Persönlichkeitstest.
+          <br /> Im Gegensatz zu Tests wie z.B. „16 Personalities“, die jeden
+          Menschen in starre Persönlichkeitstypen zwängen wollen, erkennen wir
+          an, dass jeder Mensch individuell ist. Deshalb messen wir
+          Übereinstimmungen zu einzelnen „Persönlichkeitsstilen“.
+          <br />
+          Persönlichkeitsstile sind Muster aus stabilen Merkmalen in Menschen,
+          die sich aus der individuellen Lebensgeschichte ergeben. Youlity hilft
+          Ihnen, ein differenziertes Modell Ihrer individuellen Persönlichkeit
+          zu erstellen.
+        </p>
+      </Collapse>
+      <Card>
+        <BrandHeadline firstWord="Your" secondWord="Personality" />
+        <SmallImg src="/img/style_info.svg" alt="" />
+        <h2>Der Youlity-Test</h2>
+        <BulletList listType="bullet_circle">
+          <li>Entdecken Sie Ihre individuellen Persönlichkeitsstile.</li>
+          <li>
+            Entwickeln Sie ein besseres Verständnis für sich selbst in
+            zwischenmenschlichen Beziehungen.
+          </li>
+          <li>
+            Profitieren Sie von speziell auf Sie zugeschnittenen Alltagstipps.
+          </li>
+        </BulletList>
+        <Link to="/motives/intro">
+          <Button>Test starten</Button>
```

Comment:
I would remove the Button and style the Link instead

Reply:
I thought about that too, but then I would have duplicate styles for the "button"-button and the "link"-Button. So I think this is cleaner, even if I have to wrap the button in the Link-component.